flight/Modules/Sensors/simulated/simsensors.c

```diff
@@ -241,6 +241,7 @@ static void SensorsTask(void *parameters)
 
 static void simulateYasim()
 {
+#if !(defined(_WIN32) || defined(WIN32) || defined(__MINGW32__))
```

Comment:
I see this combo a lot. We should probably have a macro somewhere so it's maintainable (and actually, our Makefiles already determine this right?).